packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss

```diff
@@ -41,11 +41,15 @@ cxl-vaadin-accordion {
       word-break: break-word;
     }
 
-    cxl-accordion-card[opened] {
-      .entry-title {
-        overflow: visible;
+    cxl-accordion-card { display: table; margin: 0;
```

Comment:
We are most certainly not doing `display: table`

Reply:
ok ill find an alternative.